app/services/collections/delete.rb

```diff
@@ -0,0 +1,40 @@
+module Collections
+  class Delete
+    
+    def self.call(collection_id, user)
+      new(collection_id, user).call
+    end
+  
+    def initialize(collection_id, user)
+      @collection = Collection.find(collection_id)
+      @user = user
+    end
+
+    def call
+      delete_collection
+      update_statistics
+      update_user_statistics
+    end
+
+    private
+
+    attr_reader :user, :collection
+
+    def delete_collection
+      CollectionWebsite.where("collection_id = ?", collection.id).destroy_all
+      collection.destroy
+    end
+
+    def update_statistics
+      Statistic.update_statistics(sites: CollectionWebsite.count)
+      Statistic.update_statistics(lists: Collection.count)
+    end
```

Comment:
You can pass in a single hash, like this:

```ruby
Statistic.update_statistics(
  sites: CollectionWebsite.count
  lists: Collection.count)
```

This will result in a single update, rather than sending two separate updates.